### PR TITLE
Reset password flow

### PR DESCRIPTION
**Pull request by teb111**

### CODE REVIEW

File: src/user/entities/userPhoneVerification.entity.ts
Comment (abiodun-michael) [inline src/user/entities/userPhoneVerification.entity.ts:1]: This is not needed as otp will be stored in cache

File: src/user/user.module.ts [hunk 1]
```diff
@@ -6,10 +6,17 @@ import { TypeOrmModule } from '@nestjs/typeorm';
...
 
 @Module({
-  imports: [TypeOrmModule.forFeature([User, Profile])],
-  providers: [UserService, UserRepository, ProfileRepository],
+  imports: [TypeOrmModule.forFeature([User, Profile, UserPhoneVerification])],
+  providers: [
+    UserService,
+    UserRepository,
+    ProfileRepository,
+    UserPhoneVerificationRepository,
```
Comment (abiodun-michael) [inline src/user/user.module.ts:18]: Kindly remove this

**Review by abiodun-michael — CHANGES_REQUESTED** [review]

File: src/notification/sms/sms.service.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { ProviderService } from '../../provider/provider.service';
+
+@Injectable()
+export class SmsService {
+  constructor(
+    private readonly africasTalkingService: AfricaTalkingService,
+    private readonly providerService: ProviderService,
+  ) {}
+
+  async sendOtpSms(to: string[], message: string, from: string) {
```
[the remark this answers is not in the document]
Comment (teb111) [reply src/notification/sms/sms.service.ts:12]: what do you mean here?

File: src/provider/entities/providerRequestResponse.entity.ts
[the remark this answers is not in the document]
Comment (teb111) [reply src/provider/entities/providerRequestResponse.entity.ts:1]: for storing logs made i will need it when working on verification

File: src/auth/auth.module.ts [hunk 3]
```diff
@@ -5,10 +5,25 @@ import { EncryptionService } from 'src/utils/encryption.service';
 import { UserModule } from 'src/user/user.module';
 import { JwtService } from 'src/utils/jwt.service';
 import { AuthController } from './auth.controller';
+import { SmsService } from 'src/notification/sms/sms.service';
+import { AfricaTalkingService } from 'src/notification/sms/africatalking.service';
```
[the remark this answers is not in the document]
Comment (teb111) [reply src/auth/auth.module.ts:9]: it's not compling if you do not add it

**Pull request closed without merging**